modules/sdk-coin-atom/src/lib/transaction.ts

```diff
@@ -125,6 +125,8 @@ export class Transaction extends BaseTransaction {
   }
 
   serialize(): string {
+    // TODO - Check if we can get the pub key from the encoded/decoded tx data
```

Comment:
add a ticket for it plz

Reply:
this is a draft PR, consulting with cosmos team on how we wish to proceed